Functions.Templates/Templates/DurableFunctionsOrchestrator-CSharp/DurableFunctionsOrchestratorCSharp.cs

```diff
@@ -0,0 +1,37 @@
+/*
+ * Before running this sample, please create a Durable Activity function (default name is "hello")
+ */
+
+#if (portalTemplates)
+#r "Microsoft.Azure.WebJobs.Extensions.DurableTask"
+
+public static async Task<List<string>> Run(DurableOrchestrationContext context)
+#endif
+#if (vsTemplates)
+using System.Collections.Generic;
+using System.Threading.Tasks;
+using Microsoft.Azure.WebJobs;
+
+namespace Company.Function
+{
+    public static class DurableFunctionsOrchestratorCSharp
+    {
+        [FunctionName("DurableFunctionsOrchestratorCSharp")]
+        public static async Task<List<string>> Run(
+            [OrchestrationTrigger] DurableOrchestrationContext context)
+#endif
+        {
+            var outputs = new List<string>();
+
+            // "hello", the first parameter to CallFunctionAsync, is the name of your Activity Function.
```

Comment:
>CallFunctionAsync [](start = 47, length = 17)

Replace "CallFunctionAsync" with "CallActivityAsync". This method was renamed in the latest bits.